contracts/modules/interfaces/IGoldenEggMetadata.sol

```diff
@@ -9,6 +9,54 @@ import { ISoundEditionV1 } from "@core/interfaces/ISoundEditionV1.sol";
  * @notice The interface for the Sound Golden Egg metadata module.
  */
 interface IGoldenEggMetadata is IMetadataModule {
+    // =============================================================
```

Comment:
can't change already deployed interface, this should be a new interface

Reply:
should extend and make new one for this

Reply:
Ah, screw it, let's make a new contract and duplicate the tests.